assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -75,11 +75,11 @@ export const TIPS = {
     ],
   },
   addLinks: {
-    title: __('Add links to design elements', 'web-stories'),
+    title: __('Add links to Story elements', 'web-stories'),
     figureSrc: 'images/help-center/add_link_module_6',
     description: [
       __(
-        'Go to the <strong>Links</strong> section under the Design tab and enter a URL. To remove a link, select the "X"<screenreader> (Clear)</screenreader> button.',
+        'Go to the Design tab. Scroll down to <strong>Links</strong> and enter a URL. To remove a link, select the "X" (clear) button next to the URL box.',
```

Comment:
Do we not need the `<screenreader>` element anymore?

Reply:
good catch! Yes that should still be there, that is so that `(clear)` only shows up in screen reader. That's my bad for not including that in the ticket.